pytest_bdd/feature.py

```diff
@@ -250,6 +250,8 @@ def __bool__(self):
 class Feature(object):
 
     """Feature."""
+    strict_gherkin = None
```

Comment:
I don't think that the Feature class should contain configuration options like `strict_gherkin` or `base_dir`. Instead, callers of `Feature(...)` should consult the current config from the stack.